src/LibChorus/Model/ServerSettingsModel.cs

```diff
@@ -31,6 +30,7 @@ public static string LanguageForgeServer
 			}
```

Comment:
Nothing new, but I find it confusing that `LanguageForgeServer` returns a name with leading dot. Also, we should check that `lfServer` starts with a dot.

Reply:
I'll add a comment. It needs to include the leading dot so it can be replaced with the leading hyphen in -qa.lf.org